src/metadata/metadata_raw.h

```diff
@@ -23,6 +23,7 @@ enum ocf_metadata_raw_type {
 	 * flushing to/landing from SSD
 	 */
 	metadata_raw_type_ram = 0,
+	metadata_raw_type_ram_pmem = 0,
```

Comment:
The enum value is repeated. I wonder how `IRAW` array initializer will behave in this case.